### PR TITLE
Buffs Forcewall

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: The forcewall spell now creates a 3x1 wall which the caster can pass through.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
the buff it deserves

**Comment by kevinz000** [comment]
nerf blink aka manual rta

File: code/modules/spells/spell_types/forcewall.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		new /obj/effect/forcefield/wizard(get_step(user, SOUTH),user)
+
+
+/obj/effect/forcefield/wizard
+	var/mob/wizard
+
+/obj/effect/forcefield/wizard/New(atom/loc, mob/summoner)
+	..()
+	wizard = summoner
+	spawn(300)
```
Comment (Xhuis) [inline code/modules/spells/spell_types/forcewall.dm]: ADDTIMEEEEEEEEEEEEEEEEEEEER

Comment (ChangelingRain) [inline code/modules/spells/spell_types/forcewall.dm]: NOT EVEN THAT!
`QDEL_IN(src, 300)`!

**Comment by KorPhaeron** [comment]
@ChangelingRain 

okay

**Comment by ExcessiveUseOfCobblestone** [comment]
kevinz not liking a spell that allows you to do what old RTA did, which he wore religiously.

really makes you think.

**Comment by Shadowlight213** [comment]
@ExcessiveUseOfCobby he just doesn't want anyone else but him having it.

**Pull request merged**